lib/reporters/base.js

```diff
@@ -168,11 +168,17 @@ exports.list = function(failures){
     var err = test.err
       , message = err.message || ''
       , stack = err.stack || message
-      , index = stack.indexOf(message) + message.length
-      , msg = stack.slice(0, index)
+      , index = stack.indexOf(message)
       , actual = err.actual
       , expected = err.expected
       , escape = true;
+    if (index === -1) {
+      msg = message;
+    } else {
+      index += message.length;
+      msg = stack.slice(0, index);
+      stack = stack.slice(index + 1)
```

Comment:
This is unnecessary and it eating the stack since the code is already doing it at line `208`: https://github.com/mochajs/mocha/pull/1631/files#diff-5c8a5788ecd4b4af2a29bc27ac4ae986R208